code/datums/flock/flockunlockable.dm

```diff
@@ -41,7 +41,7 @@ ABSTRACT_TYPE(/datum/unlockable_flock_structure)
 	structType = /obj/flock_structure/relay
 
 	check_unlocked()
-		return ..() || (src.my_flock.total_compute() >= FLOCK_RELAY_COMPUTE_COST && !src.my_flock.relay_in_progress && !src.my_flock.relay_finished)
+		return ..() || (src.my_flock.total_compute() >= FLOCK_RELAY_COMPUTE_COST && !src.my_flock.relay_in_progress && !src.my_flock.relay_finished && !src.my_flock.flockmind.tutorial)
```

Comment:
can we split this into at least one var, maybe the relay progress and finished bits - or put this on multiple lines, hard 2 read